test/src/test/kotlin/graphql/nadel/tests/EngineTestHook.kt

```diff
@@ -43,6 +44,10 @@ interface EngineTestHook {
         return builder
     }
 
+    fun isSchemaValid(errors: Set<NadelSchemaValidationError>): Boolean {
```

Comment:
Hook to bypass validation e.g. to ensure we let the malformed JSW schema through.

See the test `let-jsw-do-jsw-things.yml`